package.json

```diff
@@ -37,9 +37,11 @@
     "karma-webpack": "^1.7.0",
     "mocha": "^2.4.5",
     "phantomjs-prebuilt": "^2.1.7",
-    "react": "^15.0.0",
```

Comment:
The problem with *npm link* and *react*: https://gist.github.com/jimfb/4faa6cbfb1ef476bd105#gistcomment-1686426

Also, if you use react-onclickoutside, then you will have react installed, no?